core/src/main/java/org/infinispan/stats/impl/ClusterContainerStatsFactory.java

```diff
@@ -0,0 +1,20 @@
+package org.infinispan.stats.impl;
+
+import org.infinispan.factories.AbstractComponentFactory;
+import org.infinispan.factories.AutoInstantiableFactory;
+import org.infinispan.factories.annotations.DefaultFactoryFor;
+import org.infinispan.stats.ClusterContainerStats;
+
+/**
+ * @author Ryan Emerson
+ * @since 9.0
+ */
+@SuppressWarnings("unused")
```

Comment:
Is this needed? I don't have it in compiler, but I am surprised it gives an unused warning for a public class.